### PR TITLE
chore(contributing): change lit-html to lit

### PR DESCRIPTION
**Pull request by njfamirm**
Just change `lit-html` to `lit` in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# Contributing to lit-html
+# Contributing to lit
 
-Thank you for your interest in contributing to lit-html!
+Thank you for your interest in contributing to lit!
```
Comment (justinfagnani) [inline CONTRIBUTING.md:3]: Capitalize Lit here and the two places below.

**Review by justinfagnani — CHANGES_REQUESTED** [review]
Thanks for the fix!

We capitalize Lit though. After fixing that this look good.

**Review by justinfagnani — APPROVED** [review]

**Comment by justinfagnani** [comment]
Thanks!

**Comment by njfamirm** [comment]
> Thanks!

You're welcome

**Comment by AliMD** [comment]
👍🏻

**Pull request merged**